dev/com.ibm.ws.concurrent.persistent_fat_multiple/test-applications/persistmultitest/resources/WEB-INF/ibm-web-bnd.xml

```diff
@@ -11,6 +11,8 @@
  -->
 <web-bnd xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://websphere.ibm.com/xml/ns/javaee"
     xsi:schemaLocation="http://websphere.ibm.com/xml/ns/javaee http://websphere.ibm.com/xml/ns/javaee/ibm-web-bnd_1_0.xsd" version="1.0">
-
+	<resource-ref name="jdbc/DefaultDataSource" binding-name="jdbc/DefaultDataSource"/>
```

Comment:
these resource-refs can be removed when we go back down to 1 datasource for the bucket

Reply:
same comment applies to the ibm-web-ext.xml and the web.xml too